src/object/elo.c

```diff
@@ -350,6 +350,85 @@ elo_copy (DB_ELO * elo, DB_ELO * dest)
   return ret;
 }
 
+/*
+ * elo_copy_with_suffix () - Similar to the LOB_PERMANENT_CREATED case in elo_copy(),
+                             but called when adding a suffix to the destination path during copy.
+ * return: error code
+ * elo(in): DB_ELO structure that represents the original source file
+ * dest(out): DB_ELO structure that represents the copied file
+ * suffix(in): suffix that will be added to the destination path when copying
+ */
+int
+elo_copy_with_suffix (DB_ELO * elo, DB_ELO * dest, const char *suffix)
+{
+  int ret = NO_ERROR;
+  ES_URI out_uri, real_locator;
+  char *locator = NULL;
+  char *meta_data = NULL;
+
+  assert (elo != NULL);
+  assert (dest != NULL);
+  assert (elo->type == ELO_FBO);
+  assert (elo->locator != NULL);
+
+  /* create elo instance and copy file */
+  if (elo->meta_data != NULL)
+    {
+      meta_data = db_private_strdup (NULL, elo->meta_data);
+      if (meta_data == NULL)
+	{
+	  assert (er_errid () != NO_ERROR);
+	  return er_errid ();
```

Comment:
예외처리 루틴이 일관성 있었으면 좋겠습니다. 어떤 코드는 에러 발생 시 바로 리턴하고 어떤 코드는 error_return 레이블로 분기하면 코드 일관성이 떨어집니다.

Reply:
코드 수정을 다음과 같이 하겠습니다.
```suggestion
	  assert (er_errid () != NO_ERROR);
	  ret =  er_errid ();

          goto error_return;
```

이렇게 하면 모든 예외처리에서 error_return 레이블로 분기하게 되고 일관성이 보장됩니다.